README.md

```diff
@@ -22,3 +22,15 @@ shared_secret = ap.shared_secret
 # Extract (default) recipient id
 recipient_id = ap.recipient_id
 ```
+
+## Unit tests ##
+
+To be sure that unit tests run properly, you can run them in a docker container.
+
+```bash
+docker run -ti --rm -v $(pwd):/opt/aliquot-pay ruby:2.7.4 bash
+cd /opt/aliquot-pay
+bundle install
+bundle exec rspec
```

Comment:
I tried this out but ended up with:

```
$ docker run -ti --rm -v $(pwd):/opt/aliquot-pay ruby:2.7.4 bash
[...]
Status: Downloaded newer image for ruby:2.7.4

root@2cb4696f074f:/# cd /opt/aliquot-pay
root@2cb4696f074f:/opt/aliquot-pay# bundle install
Fetching gem metadata from https://rubygems.org/.....
Resolving dependencies...
[...]
Bundle complete! 2 Gemfile dependencies, 20 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.
root@2cb4696f074f:/opt/aliquot-pay# bundle exec rspec
/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/deprecations.rb:6:in `<module:Validation>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/messages/abstract.rb:9:in `<module:Messages>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/messages/abstract.rb:9:in `<module:Messages>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/messages/abstract.rb:9:in `<module:Messages>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/messages/abstract.rb:9:in `<module:Messages>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/messages/abstract.rb:9:in `<module:Messages>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/messages/abstract.rb:9:in `<module:Messages>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/messages/abstract.rb:9:in `<module:Messages>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/messages/abstract.rb:9:in `<module:Messages>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/type_specs.rb:6:in `class_eval' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/type_specs.rb:6:in `class_eval' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/schema/class_interface.rb:7:in `<module:Validation>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/schema/class_interface.rb:7:in `<module:Validation>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/schema/class_interface.rb:7:in `<module:Validation>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/schema/class_interface.rb:7:in `<module:Validation>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/schema/class_interface.rb:7:in `<module:Validation>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/schema/class_interface.rb:7:in `<module:Validation>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/schema/class_interface.rb:7:in `<module:Validation>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/schema/class_interface.rb:7:in `<module:Validation>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

/usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/schema/class_interface.rb:7:in `<module:Validation>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead


An error occurred while loading ./spec/generator_spec.rb.
Failure/Error: require 'aliquot'

NoMethodError:
  undefined method `size' for nil:NilClass
# /usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/schema/deprecated.rb:11:in `input_processor'
# /usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/schema/class_interface.rb:168:in `default_options'
# /usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation/schema/class_interface.rb:38:in `new'
# /usr/local/bundle/gems/dry-validation-0.12.3/lib/dry/validation.rb:23:in `Schema'
# /usr/local/bundle/gems/aliquot-2.0.0/lib/aliquot/validator.rb:81:in `<module:Validator>'
# /usr/local/bundle/gems/aliquot-2.0.0/lib/aliquot/validator.rb:9:in `<module:Aliquot>'
# /usr/local/bundle/gems/aliquot-2.0.0/lib/aliquot/validator.rb:8:in `<top (required)>'
# /usr/local/bundle/gems/aliquot-2.0.0/lib/aliquot.rb:1:in `require'
# /usr/local/bundle/gems/aliquot-2.0.0/lib/aliquot.rb:1:in `<top (required)>'
# ./spec/generator_spec.rb:1:in `require'
# ./spec/generator_spec.rb:1:in `<top (required)>'
No examples found.


Finished in 0.00002 seconds (files took 0.13371 seconds to load)
0 examples, 0 failures, 1 error occurred outside of examples
```

I'm trying to figure out why it exploded ...

Reply:
Aah, never mind. I was on the master branch :facepalm: I.e. with aliquot 2.0.0, not 2.1.0.

Reply:
On the right branch :tada: 

```
16 examples, 0 failures
```